lib/mix_test_watch/paths.ex

```diff
@@ -12,8 +12,8 @@ defmodule MixTestWatch.Path do
 
   @spec watching?(String.t) :: boolean
 
-  def watching?(path) do
-    watched_directory?( path ) && elixir_extension?( path )
+  def watching?(config \\ %{include: []}, path) do
+    _watching?( path ) || _watching?(config, path)
```

Comment:
I think this is now very confusing logic. Wouldn't it suit your use case better to be able to add more filetypes to respond to?